src/open_or_senior.py

```diff
@@ -0,0 +1,12 @@
+"""Test to see if club members are seniors."""
+
+
+def open_or_senior(data):
+    """Test to see if club members are seniors."""
+    a = list(data)
+    for i in range(len(data)):
```

Comment:
no need to iterate over the indexes here, just iterate over the values:
`for person in data:`